README.md

```diff
@@ -1,2 +1,33 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
 # Hello-Microverse
-Microverse Project
+
+> Microverse Project Day 2
+
+## Built With
+
+- HTML
+- CSS
+- Javascript
+
+## Getting Started
+
+**This project is made solely for educaitonal purposes.**
```

Comment:
- [ ] This section is supposed to tell the user how to set up your project locally and how to open it. Kindly follow this [template](https://github.com/microverseinc/readme-template) and fill out the subtitles (prerequisites, setup, install...) 👍

Reply:
Ok, I'm on it